src/_tests__/Calculator.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { render, fireEvent } from '@testing-library/react';
+/* eslint-disable */
+import Calculator from '../components/Calculator';
```

Comment:
- The same issue is applicable here.

Reply:
- The same issue is applicable to your Calculator components where you've imported the 
`import './Calculator.css';` which should be `import './calculator.css';
`